moonraker/components/file_manager/metadata.py

```diff
@@ -68,12 +69,15 @@ def _regex_find_ints(pattern: str, data: str) -> List[int]:
             pass
     return []
 
-def _regex_find_first(pattern: str, data: str) -> Optional[float]:
+def _regex_find_first(pattern: str,
+                      data: str,
+                      group: int = 1
+                      ) -> Optional[float]:
```

Comment:
I don't think the addition of a `group` here is necessary, after all this helper is intended to find the first captured group and convert it to a float.  In places where you need to group an expression you should be able use non-capturing groups.  For example:
```
(?:Plastic\sweight|Material\sWeight)
```

Reply:
You're right. I've completely forgot that non-capturing groups exist. Thanks, I've changed the code to use them.